server/routes/api.py

```diff
@@ -244,6 +244,56 @@ def post_comment(author_id: int, post_id: int) -> Response:
     return Response(status=httpStatus.OK)
 
 
+@bp.route("/authors/<int:author_id>/followers", methods=["GET"])
+def get_followers(author_id: int) -> Response:
+    followers = Requests.query.filter_by(to=author_id).all()
+    return (
+        make_response(jsonify(type="followers", items=[f.json() for f in followers])),
+        httpStatus.OK,
+    )
+
+
+@bp.route("/authors/<int:author_id>/followers/<int:follower_id>", methods=["GET"])
+def is_follower(author_id: int, follower_id: int) -> Response:
+    follower = Requests.query.filter_by(to=author_id, initiated=follower_id).first()
+    if not follower:
+        return Response(status=httpStatus.NOT_FOUND)
+    return (
+        make_response(jsonify(type="followers", items=[follower.json()])),
+        httpStatus.OK,
+    )
+
+
+@bp.route("/authors/<int:author_id>/followers/<int:follower_id>", methods=["DELETE"])
+def remove_follower(author_id: int) -> Response:
```

Comment:
This endpoint definitely needs to be protected with `login_required` and check if the author id passed is equal to the current_user

Reply:
yep I'll add a fix for it